lib/logger_json/formatters/datadog_logger.ex

```diff
@@ -0,0 +1,56 @@
+defmodule LoggerJSON.Formatters.DatadogLogger do
+  @moduledoc """
+  DataDog formatter.
```

Comment:
Please add a note that documentation for the format is here - https://docs.datadoghq.com/logs/processing/attributes_naming_convention/#default-standard-attribute-list